ExtraTasks-1.playground/Contents.swift

```diff
@@ -0,0 +1,95 @@
+//1. what digit is bigger ?
+
+let height = 15
+let width  = 20
+
+if height > width {
+    print("Height value: \(height). Height is bigger")
+} else if height < width {
+    print("Width value: \(width). Width is bigger")
+} else {
+    print("The values are the same")
+}
+
+
+/*2. Check if we take all that we need to go for a work
+Нам надо идти на работу, но нужно проверить все ли мы взяли или нет?
+Нам точно нужны ключи документы и ручка, но из еды нам надо яблоко или апельсин.
+Для решения этой задачи нам помогут логические операторы **`|| &&`**.
+*/
+
+let hasKey       = true
+let hasDocuments = true
+let hasPen       = true
+let takenApple   = false
+let takenOrange  = true
+var shouldGoToWork : Bool
+
+if hasKey && hasDocuments && hasPen && (takenApple || takenOrange) {
+    shouldGoToWork = true
+    print("Go to the work Buddy!")
+} else {
+    shouldGoToWork = false
+    print("You forgot something at home , come home ")
+}
+
+/* 3. Реализовать данные условия для числа:
+ 
+ 1. Если число делится без остатка на 5 выводим сообщение Fiz
+ 2. Если число делится без остатка на 3 выводим сообшение Buz
+ 3. Если число делится без остатка и на 3 и на 5 выводим сообшение FizBuz
+ */
+
+let anyDigit = 22
+
+if  anyDigit % 5 == 0 && anyDigit % 3 == 0{
```

Comment:
We should have empty space before and after `{` so please add it before `{`. Also, I think you have an extra space after `if`